src/server/controllers/messageController.js

```diff
@@ -0,0 +1,43 @@
+const db = require('../controllers/messageController');
+
+const messageController = {};
+
+messageController.postMessage = (req, res, next) => {
+  const { message_id, message, created_at } = req.body;
+
+  const query = `
+  INSERT INTO Messages (message_id, message, created_at)
```

Comment:
We just need to make sure the table names and column names match up. I think the messages table is actually 'public.messages' and the columns it's expecting are named a bit differently. Other than that I think everything looks great!